src/styles/blocks/card.scss

```
@@ -0,0 +1,131 @@
+$w-cont: #fff;
+$color-b: #060b35;
+$cont-color: #616070;
+$blue-cont: #00acdc;
+
+html {
+  font-family: "Roboto", Helvetica, sans-serif;
+  font-style: normal;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  width: 200px;
+  border-radius: 5px;
+  background-color: #{$w-cont};
+  border: 1px solid #f3f3f3;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  padding: 0 16px;
+  padding-top: 32px;
+  padding-bottom: 16px;
```

use padding shorthand
```suggestion
  padding: 32px 16px 16px;
```